test/EndToEndTests/Tests/Client/Build.Desktop/ContainmentTest/ContainmentTest.cs

```diff
@@ -697,9 +697,16 @@ public void QueryEntityAndExpandContainedEntity()
         {
             Dictionary<string, int[]> testCases = new Dictionary<string, int[]>()
             {
-                { "Accounts(101)?$expand=MyGiftCard", new int[] {2, 4} },
-                { "Accounts(101)?$expand=MyPaymentInstruments", new int[] {4, 15} },
-                { "Accounts(101)?$select=AccountID&$expand=MyGiftCard($select=GiftCardID)", new int[] {2, 1}  }
+                { "Accounts(101)?$select=AccountInfo/FirstName, AccountInfo/LastName", new int[] {1, 0, 0} },
+                { "Accounts(101)?$select=AccountID&$expand=MyPaymentInstruments($select=PaymentInstrumentID;$expand=TheStoredPI)", new int[]{7,4,4} },

```

Comment:
>                { "Accounts(101)?$select=AccountID&$expand=MyPaymentInstruments($select=PaymentInstrumentID;$expand=TheStoredPI) [](start = 0, length = 128)

Do we have any test cases that test multiple nested levels of expand? i.e., $select=AccountID&$expand=MyPaymentInstruments($select=PaymentInstrumentID;$expand=TheStoredPI($select=someproponstoredpi))?